docs/rationale.md

```diff
@@ -54,6 +54,19 @@ Here are a few examples of things that are out of scope for prettier:
 * Adding/removing `{}` and `return` where they are optional.
 * Turning `?:` into `if then else`.
 
+## Can prettier add an option to do X?
+
+A lot of the current options in prettier come from the `recast` options of which prettier is a fork, and some of them were added later due to popular demand. Ideally we would not want options in prettier at all to keep printing things consistently and have less maintenance overhead but the JavaScript community has already chosen a few different coding styles people are used to:
```

Comment:
> A lot of the current options in prettier come from the `recast` options of which prettier is a fork

This is not true. James Long didn't want any options and I wanted for prettier to work for Facebook but he didn't want to make the Facebook styleguide the default one, so we started adding a few options to make it work for Facebook.

Reply:
Ops I read someone saying that and thought it was true